code-challenges.js

```diff
@@ -23,12 +41,25 @@ var middleLetters1 = "hello"
 var middleLetters2 = "rhinoceros"
 // Expected output: “oc”
 
-
-
+const middle = str => {
+    let newStr = ""
+    if(!(str.length %2)){
+        newStr = str.slice(str.length/2 -1,str.length/2 +1)
+    } else if(str.length %2){
+        newStr = str.slice(str.length/2,str.length/2 +1)
+    }
+    return newStr
+}
+console.log(middle(middleLetters1))
+console.log(middle(middleLetters2))
 
 // --------------------4) Create a class to get the area of a sphere. Create three spheres with different radi as test cases. Area of a sphere =  4πr^2 (four pi r squared)
 
-
+const areaSphere = r => {
+    return 4*Math.PI*Math.pow(r,2)
+}
```

Comment:
This is great logic as a function but now the next step would be to use a class!

Reply:
didn't read that right! thanks!